docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,552 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fe59d027-121a-4411-9dc6-0c1756f205e2",
+   "metadata": {},
+   "source": [
+    "# Alert activations\n",
+    "\n",
+    "There is a way to get all activations of alerts. It can be done in three ways:\n",
```

Comment:
## List alert activations 

  "There are three ways to retrieve the list of alert activations:\n",
    "- From run_db\n",
    "- For a specific project\n",
    "- For a specific alert config"